packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -2145,7 +2150,6 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
       // `selection` is the only change.
       _handleSelectionChanged(value.selection, (_textInputConnection?.scribbleInProgress ?? false) ? SelectionChangedCause.scribble : SelectionChangedCause.keyboard);
     } else {
-      hideToolbar();
```

Comment:
Could you double check deleting this is safety? @LongCatIsLooong THX : )

Reply:
In UIKit/Android apps, the text edit menu should go away once the user starts typing. Would this change that behavior?

Reply:
Will add a test to cover this.

Reply:
Ah `_handleSelectionChanged` is called in `_formatAndSave` too, that seems to be a messy callpath. Do you mind adding a comment to explain why we're not hiding the handles here (or link the issue)?

Reply:
done